lineman/app/components/thread_page/activity_card/new_comment.haml

```diff
@@ -17,9 +17,7 @@
       %h3.new-comment__in-reply-to{ng-if: 'comment.parentId', id: 'event-{{event.id}}'}
         %strong
           %a{lmo-href-for: 'comment.author()'}> {{ comment.authorName() }}
-        %span{translate: 'new_comment_item.in_reply_to'}
-        %strong
-          %a{lmo-href-for: 'comment.parentAuthor()'}> {{ comment.parentAuthorName }}
+        %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient: '{{comment.parentAuthorName}}'}
```

Comment:
Active model serializer wouldn't do it.

On Fri, 18 Dec 2015 at 15:51 James Kiesel <notifications@github.com> wrote:

> In lineman/app/components/thread_page/activity_card/new_comment.haml
> <https://github.com/loomio/loomio/pull/2836#discussion_r47989632>:
>
> > @@ -17,9 +17,7 @@
> >        %h3.new-comment__in-reply-to{ng-if: 'comment.parentId', id: 'event-{{event.id}}'}
> >          %strong
> >            %a{lmo-href-for: 'comment.author()'}> {{ comment.authorName() }}
> > -        %span{translate: 'new_comment_item.in_reply_to'}
> > -        %strong
> > -          %a{lmo-href-for: 'comment.parentAuthor()'}> {{ comment.parentAuthorName }}
> > +        %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient: '{{comment.parentAuthorName}}'}
>
> What's the reasoning for not serializing out the parent with the comment?
> I seem to remember we used to do that... was it too slow?
> … <#msg-f:1520862095851779723_>
> On Fri, Dec 18, 2015 at 3:48 AM, Robert Guthrie <notifications@github.com>
> wrote: In lineman/app/components/thread_page/activity_card/new_comment.haml
> <#2836 (comment)
> <https://github.com/loomio/loomio/pull/2836#discussion_r47989490>>: > @@
> -17,9 +17,7 @@ > %h3.new-comment__in-reply-to{ng-if: 'comment.parentId',
> id: 'event-{{event.id}}'} > %strong > %a{lmo-href-for:
> 'comment.author()'}> {{ comment.authorName() }} > - %span{translate:
> 'new_comment_item.in_reply_to'} > - %strong > - %a{lmo-href-for:
> 'comment.parentAuthor()'}> {{ comment.parentAuthorName }} > +
> %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient:
> '{{comment.parentAuthorName}}'} hrrmm.. yes I guess so. would be nice to
> have the whole user object though. I wonder if we can do that with AMS yet.
> … <#151b2fd5018add38_> On Fri, 18 Dec 2015 at 15:15 James Kiesel <
> notifications@github.com> wrote: In
> lineman/app/components/thread_page/activity_card/new_comment.haml <#2836
> <https://github.com/loomio/loomio/pull/2836> (comment) <#2836 (comment)
> <https://github.com/loomio/loomio/pull/2836#discussion_r47987727>>>: > @@
> -17,9 +17,7 @@ > %h3.new-comment__in-reply-to{ng-if: 'comment.parentId',
> id: 'event-{{event.id}}'} > %strong > %a{lmo-href-for:
> 'comment.author()'}> {{ comment.authorName() }} > - %span{translate:
> 'new_comment_item.in_reply_to'} > - %strong > - %a{lmo-href-for:
> 'comment.parentAuthor()'}> {{ comment.parentAuthorName }} > +
> %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient:
> '{{comment.parentAuthorName}}'} Oh, I see. specifically the case where
> we're jumping directly to a comment and not back-loading it's parent.
> Should we be serializing out the parentAuthorUsername alongside the
> parentAuthorName then? … <#msg-f:1520859769167276480_> On Fri, Dec 18, 2015
> at 3:11 AM, Robert Guthrie <notifications@github.com> wrote: In
> lineman/app/components/thread_page/activity_card/new_comment.haml <#2836
> <https://github.com/loomio/loomio/pull/2836> <#2836
> <https://github.com/loomio/loomio/pull/2836>> (comment) <#2836
> <https://github.com/loomio/loomio/pull/2836> (comment) <#2836 (comment)
> <https://github.com/loomio/loomio/pull/2836#discussion_r47987588>>>>: >
> @@ -17,9 +17,7 @@ > %h3.new-comment__in-reply-to{ng-if: 'comment.parentId',
> id: 'event-{{event.id}}'} > %strong > %a{lmo-href-for:
> 'comment.author()'}> {{ comment.authorName() }} > - %span{translate:
> 'new_comment_item.in_reply_to'} > - %strong > - %a{lmo-href-for:
> 'comment.parentAuthor()'}> {{ comment.parentAuthorName }} > +
> %span{translate: 'new_comment_item.in_reply_to', translate-value-recipient:
> '{{comment.parentAuthorName}}'} I don't understand what you're saying. We
> don't serialize the parent author with the comment, so there are cases when
> the parent author is not loaded in the client so we don't know their user
> id — Reply to this email directly or view it on GitHub <
> https://github.com/loomio/loomio/pull/2836/files#r47987588>. — Reply to
> this email directly or view it on GitHub <
> https://github.com/loomio/loomio/pull/2836/files#r47987727>. — Reply to
> this email directly or view it on GitHub <
> https://github.com/loomio/loomio/pull/2836/files#r47989490>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2836/files#r47989632>.
>